docs/collections.md

```diff
@@ -126,61 +109,100 @@ function formatDate(date) {
 </html>
 ```
 
-Let’s walk through some of the key parts:
-
-- `export let collection`: this is important because it exposes a prop to the page for Astro to return with all your data loaded. ⚠️ **It must be named `collection`**.
```

Comment:
Doesn’t have to be here exactly, but I think this is an important point to reiterate: this prop must be exported to work